src/main/target/MATEKF405/target.h

```diff
@@ -176,6 +176,9 @@
 
 #define PITOT_I2C_BUS               DEFAULT_I2C_BUS
 
+#define USE_OPTICAL_FLOW
+#define USE_OPFLOW_MSP
```

Comment:
`USE_OPFLOW_MSP` is already defined for all targets having more than 256k flash, it is not needed.